apis/Google.Cloud.Storage.V1/Google.Cloud.Storage.V1/StorageClientBuilder.cs

```diff
@@ -203,8 +200,10 @@ private Dictionary<string, string> GetEmulatorEnvironment(
                     }
                     // When the settings *only* support the use of an emulator, the other properties shouldn't be set.
                     CheckNotSet(BaseUri, nameof(BaseUri));
+#pragma warning disable CS0618 // Temporarily disable warnings for obsolete methods. See b/453009677 for more details.
```

Comment:
We don't have to address this in b/453009677. There's nothing else we can do here until the deprecated properties are removed from the client builder base.